go-controller/pkg/ovn/base_network_controller_policy.go

```diff
@@ -477,32 +488,44 @@ func getARPAllowACLName(ns string) string {
 	return joinACLName(ns, arpAllowPolicySuffix)
 }
 
+// Note that the hashName argument is the portGroup's Name without network scope prefix
+func (ncni *NetworkControllerNetInfo) buildPortGroup(hashName, name string, ports []*nbdb.LogicalSwitchPort, acls []*nbdb.ACL) *nbdb.PortGroup {
+	var externalIds map[string]string
+	pgName := ncni.GetNetworkScopedName(hashName)
+	externalIds = map[string]string{"name": name}
```

Comment:
I don't think we need. we have network specific external-ids

Reply:
Not because we need it, it just because that is the PG name and clearly there shouldn't be multiple PGs with the same name, just to be consistent.

Reply:
i had the same question, but we don't really use externalIDs for port groups, and if we change the value it will be more difficult to sync port groups to the new ids in the future (now we will have the same "name" for any controller type, and we can get owner controller from types.NetworkExternalID without extra parsing)

Reply:
hmm okay!